sources/effects.c

```diff
@@ -85,7 +85,7 @@ int write_oriented_tap_sequence(int fd, int numLocations, ...) {
     if (action == SLEEP) {
       usleep(50000);
     } else if (action == LONG_SLEEP) {
-      usleep(200000);
+      usleep(300000);
```

Comment:
I guess you tested it with `200000` and it was a bit flaky?

Reply:
Yeah. What do you think about making this configurable?

Reply:
Perhaps that's the best solution, yeah

Reply:
Perhaps that's the best solution, yeah

Reply:
I've reverted this change for now.